### ISSUE TITLE
utils.py throws assertion error in _load_code

### ISSUE THREAD

**Issue by coyotlinden** [pyinstaller/pyinstaller#2093]
Given the CMake line:

```
    add_custom_command(
    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}/<EXE>
    COMMAND ${PYTHON_DIRECTORY}/Scripts/pyinstaller.exe
    ARGS
       --onefile
       --log-level WARN
       --distpath ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}
            ${CMAKE_SOURCE_DIR}/<more path>/<EXE>
    COMMENT "Performing pyinstaller compile of <EXE>"
```

Which runs fine under 3.1.1, the following occurs in 3.2:

```
[21:37:54][LogScan] 21> File "C:\Python27\lib\site-packages\PyInstaller\building\utils.py", line 521, in _load_code
[21:37:54][LogScan] 21> assert loader and hasattr(loader, 'get_code')
[21:37:54][LogScan] 21> AssertionError
[21:37:54][LogScan] 21>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V120\Microsoft.CppCommon.targets(170,5): error MSB6006: "cmd.exe" exited with code 1.
```

Python 2.7.9
Windows 7

**Comment by bjones1** [comment]
This might be fixed in 7e0bb3fdeb520af4ba5e9a74c139007de570f6ae. Would you try latest version from git and report back?

**Comment by coyotlinden** [comment]
I've asked the build team to try it on a test host, will report back when I hear from them.

**Comment by coyotlinden** [comment]
Using the tarball found at https://github.com/pyinstaller/pyinstaller/zipball/develop , we untarred, built and deployed it to a build host.

This one succeeded!

**Comment by Icon0clast** [comment]
Was having the same issue, can confirm the version @coyotlinden mentioned fixed the issue.

**Comment by bjones1** [comment]
Great!

**Comment by tiangolo** [comment]
When will the fix from https://github.com/pyinstaller/pyinstaller/commit/7e0bb3fdeb520af4ba5e9a74c139007de570f6ae that @bjones1 referenced be available in a release?

...I just got bitten by the same `assert` error.

**Comment by CaseyMcGinnis** [comment]
I have gotten this error this morning as well. Tried with a very simple python file with tkinter.

**Comment by tiangolo** [comment]
While this gets fixed, I'm just using PyInstaller version 3.1, with:

```bash
pip install pyinstaller==3.1
```

**Comment by crackerjak** [comment]
I'm also awaiting the release of this fix.
